### PR TITLE
AKO-Operator 1.7.1: Adding fields to AKO Config CRD

### PR DESCRIPTION
**Pull request by akshayhavile**
Please note: AKO-Operator currently do not support helm install. Still changes has been added considering future alternative option.

### CODE REVIEW

**Review by swathinsankaran — APPROVED** [review]
LGTM

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

File: ako-operator/config/crd/bases/ako.vmware.com_akoconfigs.yaml [hunk 1]
```diff
@@ -91,6 +91,11 @@ spec:
                       labelValue:
                         type: string
                     type: object
+                  primaryInstance:
+                    description: PrimaryInstance defines AKO instance is primary or
```
Comment (apalsule) [inline ako-operator/config/crd/bases/ako.vmware.com_akoconfigs.yaml:95]: dont you have a default here ?

**Comment by apalsule** [comment]
Is this teh only field we added to AKO config ?

File: ako-operator/config/crd/bases/ako.vmware.com_akoconfigs.yaml (hunk 1 above)
Comment (akshayhavile) [reply ako-operator/config/crd/bases/ako.vmware.com_akoconfigs.yaml:95]: For `boolean` fields, we are not specifying default.

**Comment by akshayhavile** [comment]
> Is this teh only field we added to AKO config ?

@apalsule: Yes. As we are not publishing MCI feature, so I have not included `enableMCI` field in code.

**Pull request merged**